src/__tests__/ParserComponent.test.tsx

```diff
@@ -84,9 +85,47 @@ describe('<ParserComponent /> Unit Tests', () => {
     });
   });
 
+  describe('createCountingObjects()', () => {
+    const pi: ParserInterface = {prompt: 'test', fileType: FileType.csv};
```

Comment:
If we were to add additional test cases to this describe, the variable `pi` would change on each iteration which would cause test cases to fail if they depend on the state being the same at the start.

Consider using `beforeEach()`: https://jestjs.io/docs/en/setup-teardown